src/widgets/viewsplit.cpp

```diff
@@ -711,6 +715,16 @@ void ViewSplit::createContextMenuOnTabBar(QMenu *p_menu, int p_tabIdx)
     }
 }
 
+void ViewSplit::newTab(int p_idx)
```

Comment:
The name here is confusing. Rename it to `CreateQuickNote` without the `p_idx`.

Reply:
Done.